Sources/Mini/ActionReducer.swift

```diff
@@ -1,35 +1,19 @@
-/*
- Copyright [2019] [BQ]
-
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-
- http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
- */
-
 import Foundation
-import RxSwift
-
+import Combine
 /**
  The `Reducer` defines the behavior to be executed when a certain
  `Action` object is received.
  */
-public class Reducer<A: Action>: Disposable {
+ @available(iOS 13.0, *)
```

Comment:
Remove call `@available` annotations and instead bump the minimum target version in the `Package.swift` file.

Reply:
Fixed, now target version is 13.0.